installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3072,4 +3072,10 @@ INSERT IGNORE INTO pac_config_properties (`cfkey`,`value`,`application`,`profile
 
   update pac_v2_ui_options set optionName ="Source" where optionId='32';
   update pac_v2_ui_options set optionName ="Source" where optionId='57';
-  update pac_v2_ui_options set optionName ="Source" where optionId='43';
\ No newline at end of file
+  update pac_v2_ui_options set optionName ="Source" where optionId='43';
+
+insert ignore into cf_PolicyParams (policyID,paramKey,paramValue) values ('Azure_Check_Public_Access_For_Storage_Account','fixKey','storage-account-public-access-auto-fix');
+insert ignore into cf_PolicyParams (policyID,paramKey,paramValue) values ('UnrestrictedSqlDatabaseAccessRule_version-1','fixKey','unrestricted-sql-access-auto-fix');
+insert ignore into cf_PolicyParams (policyID,paramKey,paramValue) values ('Azure_Security_Groups_with_RDP_port_3389_should_not_be_publicly_accessible','fixKey','public-access-nsg-auto-fix');
+
+update cf_PolicyTable set fixType='silent', autoFixAvailable='true', autoFixEnabled='true' where policyId in ('Azure_Check_Public_Access_For_Storage_Account','Azure_Security_Groups_with_RDP_port_3389_should_not_be_publicly_accessible','UnrestrictedSqlDatabaseAccessRule_version-1');
```

Comment:
by default, these need to be disabled right?

Reply:
updated it to false.